baseplate/ratelimit.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import division
+
+import time
+
+
+class RateLimitExceededException(Exception):
+    """This exception gets raised whenever a rate limit is exceeded.
+    """
+    pass
+
+
+class RateLimiter:
```

Comment:
Since baseplate also supports python2, you should include the (object) here (you don't need it in python3 but it changes some behavior when you don't have it in python2).